dcgm/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,14 @@
+name: dcgm
+files:
+- name: dcgm.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/openmetrics
+  - template: instances
+    options:
+    - template: instances/openmetrics
+      overrides:
+        openmetrics_endpoint.value.example: http://localhost:9400/metrics
+        openmetrics_endpoint.description: |
+          Set this to the port of the DCGM Exporter exposing the metrics in the OpenMetrics format.
```

Comment:
I think we should decide on how we spell "dcgm exporter". Either the way it's here, or "dcgm-exporter". Maybe double-check how they themselves spell it?

Reply:
Updated to dcgm-exporter ✅